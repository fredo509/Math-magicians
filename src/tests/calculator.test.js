import React from 'react';
import { createRoot } from 'react-dom/client';
// eslint-disable-next-line
import renderer from 'react-test-renderer';
// eslint-disable-next-line
import theCalculator from '../components/calculator';

it('renders without crashing', () => {
  const div = document.createElement('div');
  const root = createRoot(div);
  root.render(<theCalculator />);
});

test('match home snapshot', () => {
  const tree = renderer.create(<theCalculator />).toJSON();
  expect(tree).toMatchSnapshot();
});
