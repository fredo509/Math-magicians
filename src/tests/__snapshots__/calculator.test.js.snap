// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match home snapshot 1`] = `<theCalculator />`;
